strings/ion_laws.json

```diff
@@ -943,8 +925,6 @@
         "PETES",
         "RATVARIANS",
         "XENOS",
-        "FETISHES",
-        "MEMES",
```

Comment:
Muh memes! Don't take away muh memes!

Reply:
We all know AIs will be memeing anyway, regardless of laws @taukausanake